tsconfig.json

```diff
@@ -1,5 +1,10 @@
 {
   "compilerOptions": {
+    "baseUrl": "./packages",
+    "paths": {
+      "graphql-tool-utilities/lib/*": ["./graphql-tool-utilities/src/*"],
+      "graphql-tool-utilities": ["./graphql-tool-utilities/src"]
```

Comment:
this is required to satisfy the lerna TypeScript mono-repo setup (when inter-project dependencies are involved). Based on the [official guidance](https://github.com/Quramy/lerna-yarn-workspaces-example#resolve-dependencies-as-typescript-modules), and modified slightly because we access `lib` submodules in our case.

Reply:
Yep we did this in quilt too